package.json

```diff
@@ -17,7 +22,17 @@
   },
   "homepage": "https://github.com/entropic-dev/ds#readme",
   "devDependencies": {
-    "typescript": "^3.5.2"
+    "@types/node": "^12.6.0",
+    "@types/yargs": "^13.0.0",
+    "@typescript-eslint/eslint-plugin": "^1.11.0",
+    "@typescript-eslint/parser": "^1.11.0",
+    "eslint": "^6.0.1",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-prettier": "^3.1.0",
```

Comment:
Do we really need `eslint` if we're using prettier? I thought the point of prettier was to not have to use a linter as well.

Reply:
A-ha! We stubbed our toe on this on `entropic` as well: prettier is only a formatter & won't lint for you. You have to bring in eslint to do the linting step.

Reply:
@chrisdickinson is correct.